Scheduler/feature_scheduler/maintel/fbs_config_sit_survey_block_t189.py

```diff
@@ -0,0 +1,194 @@
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import rubin_scheduler.scheduler.basis_functions as bf
+import rubin_scheduler.scheduler.detailers as detailers
+from lsst.ts.scheduler.utils.test.feature_scheduler_sim import MJD_START
+from rubin_scheduler.scheduler.model_observatory import ModelObservatory
+from rubin_scheduler.scheduler.schedulers import CoreScheduler
+from rubin_scheduler.scheduler.surveys import GreedySurvey
+from rubin_scheduler.scheduler.utils import Footprint, SkyAreaGenerator
+
+
+def gen_greedy_surveys(
+    nside=32,
+    nexp=1,
+    nexp_override=None,
+    exptime=30.0,
+    exptime_override=None,
+    filters=["g", "r", "z"],
+    camera_rot_limits=[-80.0, 80.0],
+    shadow_minutes=60.0,
+    max_alt=76.0,
+    moon_distance=30.0,
+    ignore_obs="DD",
+    footprint_weight=0.3,
+    slewtime_weight=3.0,
+    stayfilter_weight=3.0,
+    footprints=None,
+    seed=42,
+):
+    """This function heavily borrows from a function of the same name
+    in fbs_config_sit_survey_block_t87.py. There are mild modifications
+    to support different exposures for different filters.
+
+    Parameters
+    ----------
+    nside : int (32)
+        The HEALpix nside to use.
+    nexp : int (1)
+        The number of exposures to use in a visit.
+    nexp_override : dict (None)
+        Key value pairs to override nexp for specific filters
+    exptime : float (30.)
+        The exposure time to use per visit (seconds).
+    exptime_override : dict (None)
+        Key value pairs to override exptime for specific filters
+    filters : list of str (['r', 'i', 'z', 'y'])
+        Which filters to generate surveys for.
+    camera_rot_limits : list of float ([-80., 80.])
+        The limits to impose when rotationally dithering the camera (degrees).
+    shadow_minutes : float (60.)
+        Used to mask regions around zenith (minutes).
+    max_alt : float (76.)
+        The maximium altitude to use when masking zenith (degrees).
+    moon_distance : float (30.)
+        The mask radius to apply around the moon (degrees).
+    ignore_obs : str or list of str ('DD')
+        Ignore observations by surveys that include the given substring(s).
+    footprint_weight : float (0.3)
+        The weight on the survey footprint basis function.
+    slewtime_weight : float (3.)
+        The weight on the slewtime basis function.
+    stayfilter_weight : float (3.)
+        The weight on basis function that tries to stay avoid filter changes.
+    seed : int (42)
+        The random generator seed.
+    """
+    # Define the extra parameters that are used in the greedy survey. I
+    # think these are fairly set, so no need to promote to utility func kwargs
+    greed_survey_params = {
+        "block_size": 1,
+        "smoothing_kernel": None,
+        "seed": seed,
+        "camera": "LSST",
+        "dither": True,
+        "survey_name": "BLOCK-T87",
```

Comment:
Should this be `BLOCK-T189` instead?

Reply:
Yeah, that  was the small fixup I force pushed. It is fixed now.